core/src/main/scala/zio/jdbc/JdbcDecoder.scala

```diff
@@ -104,13 +104,35 @@ object JdbcDecoder extends JdbcDecoderLowPriorityImplicits {
   implicit val byteDecoder: JdbcDecoder[Byte]                             = JdbcDecoder(_.getByte)
   implicit val byteArrayDecoder: JdbcDecoder[Array[Byte]]                 = JdbcDecoder(_.getBytes)
   implicit val blobDecoder: JdbcDecoder[Blob]                             = JdbcDecoder(_.getBlob)
-  implicit val dateDecoder: JdbcDecoder[java.sql.Date]                    = JdbcDecoder(_.getDate)
-  implicit val timeDecoder: JdbcDecoder[java.sql.Time]                    = JdbcDecoder(_.getTime)
-  implicit val timestampDecoder: JdbcDecoder[java.sql.Timestamp]          = JdbcDecoder(_.getTimestamp)
   implicit val uuidDecoder: JdbcDecoder[java.util.UUID] =
     // See: https://stackoverflow.com/a/56267754/2431728
     JdbcDecoder(rs => i => rs.getObject(i, classOf[java.util.UUID]), "UUID")
 
+  implicit val dateDecoder: JdbcDecoder[java.sql.Date]           = JdbcDecoder(_.getDate)
+  implicit val timeDecoder: JdbcDecoder[java.sql.Time]           = JdbcDecoder(_.getTime)
+  implicit val timestampDecoder: JdbcDecoder[java.sql.Timestamp] = JdbcDecoder(_.getTimestamp)
+
+  // These `java.time.*` decoders are copied from Quill's 'ObjectGenericTimeDecoders' trait.
+  // Note:
+  //   1. These decoders probably don't work for SQLite. Quill as a separate trait, named `BasicTimeDecoders` which seems dedicated to SQLite.
+  //   2. We deliberately decided not to support `java.time.OffsetTime`.
+  //      The reasons for this choice are detailed next to the `java.time.*` Setters implementation
```

Comment:
This a point you might want to discuss during the review.
For full details about this decision, see https://github.com/zio/zio-jdbc/pull/171/files#diff-b27fd701f6b31c373ffd9ed462f852866acd4e50d332170c99db373511c52460R386-R393